schemas/device/MRICoil.schema.tpl.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_extends": "/core/schemas/products/device.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/neuroimaging/MRICoil",
+  "properties": {
+    "activeElements": {
```

Comment:
If it can be one, change property name to "activeElement". This is the convention that we follow. Meaning only when it must be multiple, the property name may be plural.

Reply:
Exactly so I modify acordingly